File: src/main/java/me/leodev/trollingfreedomreborn/trolls/random/Burn.java
package me.leodev.trollingfreedomreborn.trolls.random;

import org.bukkit.entity.Player;
import org.bukkit.event.Listener;

public class Burn implements Listener{
    public static void Burn(Player p) {
        String p2 = p.getName();
        p.setFireTicks(1000000000);
    }
}

File: src/main/java/com/leomadrassi/trollingfreedomreborn/trolls/random/Burn.kt
package com.leomadrassi.trollingfreedomreborn.trolls.random

import org.bukkit.entity.Player
import org.bukkit.event.Listener

class Burn : Listener {
    companion object {
        fun Burn(p: Player) {
            val p2 = p.name
            p.fireTicks = 1000000000
        }
    }
}
